src/people.js

```diff
@@ -2,8 +2,22 @@
 
 /* eslint-disable */
```

Comment:
Remove the 'eslint-disable' comment to ensure that ESLint is properly checking the code.